### ISSUE TITLE
CloudFlareStream "No video formats found!"

### ISSUE THREAD

**Issue by hugovdev** [yt-dlp/yt-dlp#11477]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I have **updated yt-dlp to nightly or master** ([update instructions](https://github.com/yt-dlp/yt-dlp#update-channels))
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Spain

### Used URL

https://watch.cloudflarestream.com/e8766bfd7771e53b5dc26c76b650d91a/

### Provide a description that is worded well enough to be understood

Can't download a public CloudFlareStream video.
I get CERTIFICATE_VERIFY_FAILED warnings and then an error stating "No video formats found!"

Expected result: A video download from the provided link.

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['-vU', 'https://watch.cloudflarestream.com/e8766bfd7771e53b5dc26c76b650d91a/']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2024.11.04 from yt-dlp/yt-dlp [197d0b03b] (win_exe)
[debug] Python 3.10.11 (CPython AMD64 64bit) - Windows-10-10.0.22631-SP0 (OpenSSL 1.1.1t  7 Feb 2023)
[debug] exe versions: ffmpeg 5.1.2-essentials_build-www.gyan.dev (setts), ffprobe 5.1.2-essentials_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.21.0, brotli-1.1.0, certifi-2024.08.30, curl_cffi-0.5.10, mutagen-1.47.0, requests-2.32.3, sqlite3-3.40.1, urllib3-2.2.3, websockets-13.1
[debug] Proxy map: {}
[debug] Request Handlers: urllib, requests, websockets, curl_cffi
[debug] Loaded 1838 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Latest version: stable@2024.11.04 from yt-dlp/yt-dlp
yt-dlp is up to date (stable@2024.11.04 from yt-dlp/yt-dlp)
[CloudflareStream] Extracting URL: https://watch.cloudflarestream.com/e8766bfd7771e53b5dc26c76b650d91a/
[CloudflareStream] e8766bfd7771e53b5dc26c76b650d91a: Downloading m3u8 information
WARNING: [CloudflareStream] Failed to download m3u8 information: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1007)
[CloudflareStream] e8766bfd7771e53b5dc26c76b650d91a: Downloading MPD manifest
WARNING: [CloudflareStream] Failed to download MPD manifest: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1007)
ERROR: [CloudflareStream] e8766bfd7771e53b5dc26c76b650d91a: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
Traceback (most recent call last):
  File "yt_dlp\YoutubeDL.py", line 1625, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1781, in __extract_info
  File "yt_dlp\YoutubeDL.py", line 1840, in process_ie_result
  File "yt_dlp\YoutubeDL.py", line 2846, in process_video_result
  File "yt_dlp\YoutubeDL.py", line 1122, in raise_no_formats
yt_dlp.utils.ExtractorError: [CloudflareStream] e8766bfd7771e53b5dc26c76b650d91a: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
```

**Comment by mosaphuma** [comment]
i still meet this issue too

**Comment by hugovdev** [comment]
I am able to manually download the video.mpd file with
https://videodelivery.net/eaef9dea5159cf968be84241b5cedfe7/manifest/video.mpd
So I'm not sure what's going wrong

**Comment by hugovdev** [comment]
Playing around with Postman and GET requests to videodelivery.net don't go through but they do for cloudflarestream.com

GET https://videodelivery.net/eaef9dea5159cf968be84241b5cedfe7/manifest/video.mpd
```
<html>

<body>Object not found</body>

</html>```

GET https://cloudflarestream.com/eaef9dea5159cf968be84241b5cedfe7/manifest/video.mpd
```
<?xml version="1.0" encoding="UTF-8"?>
<MPD xmlns="urn:mpeg:dash:schema:mpd:2011" profiles="urn:mpeg:dash:profile:isoff-live:2011" type="static" mediaPresentationDuration="PT38S" minBufferTime="PT8S">
    <Period id="0">
        <AdaptationSet id="800962650" mimeType="audio/mp4" segmentAlignment="true" lang="original">
            <Representation id="449134446" audioSamplingRate="44100" bandwidth="142554" codecs="mp4a.40.2">
                <AudioChannelConfiguration schemeIdUri="urn:mpeg:dash:23003:3:audio_channel_configuration:2011" value="1"></AudioChannelConfiguration>
                <SegmentTemplate duration="172695" initialization="../../eaef9dea5159cf968be84241b5cedfe7/audio/142/init.mp4?p=eyJ0eXBlIjoiaW5pdCIsInZpZGVvSUQiOiJlYWVmOWRlYTUxNTljZjk2OGJlODQyNDFiNWNlZGZlNyIsIm93bmVySUQiOjY2MzQxNTgsImNyZWF0b3JJRCI6InVwcmlkZSIsInVzZVZPRE9URkUiOmZhbHNlLCJmcm9tTWV6emFuaW5lIjpmYWxzZSwic3RvcmFnZVByb3ZpZGVyIjoyLCJ0cmFjayI6ImFjZjUxZDAwYTlkNmNiODNmNGNhNzI1ZDZiOTM2MjI3IiwicmVuZGl0aW9uIjoiNDQ5MTM0NDQ2IiwibXV4aW5nIjoiNDk5ODMzNjE1In0&amp;s=L3jChcK8L8Kgwo0zwrTDhU_DpncjwpzCqMO0esOCC8O5wonCvRTCohMOQsOQwpBMAg" media="../../eaef9dea5159cf968be84241b5cedfe7/audio/142/seg_$Number$.mp4?p=eyJ0eXBlIjoic2VnbWVudCIsInZpZGVvSUQiOiJlYWVmOWRlYTUxNTljZjk2OGJlODQyNDFiNWNlZGZlNyIsIm93bmVySUQiOjY2MzQxNTgsImNyZWF0b3JJRCI6InVwcmlkZSIsInNlZ21lbnREdXJhdGlvblNlY3MiOjMuOTE1OTgzMTU1MTY2ODI4NSwidXNlVk9ET1RGRSI6ZmFsc2UsImZyb21NZXp6YW5pbmUiOmZhbHNlLCJzdG9yYWdlUHJvdmlkZXIiOjIsInRyYWNrIjoiYWNmNTFkMDBhOWQ2Y2I4M2Y0Y2E3MjVkNmI5MzYyMjciLCJyZW5kaXRpb24iOiI0NDkxMzQ0NDYiLCJtdXhpbmciOiI0OTk4MzM2MTUifQ&amp;s=XW0iRcO7w4zCvsO5wqbCrxo0TMO3w5bDgcKhb8Oaw7dtXMKFScKfwpHCt8OBwpJaOcOv" startNumber="1" timescale="44100"></SegmentTemplate>
            </Representation>
        </AdaptationSet>
        <AdaptationSet id="386409604" mimeType="video/mp4" segmentAlignment="true" >
            <Representation id="449134449" bandwidth="405430" codecs="avc1.42c015" frameRate="30/1" height="240" width="426">
                <SegmentTemplate duration="120000" initialization="../../eaef9dea5159cf968be84241b5cedfe7/video/240/init.mp4?p=eyJ0eXBlIjoiaW5pdCIsInZpZGVvSUQiOiJlYWVmOWRlYTUxNTljZjk2OGJlODQyNDFiNWNlZGZlNyIsIm93bmVySUQiOjY2MzQxNTgsImNyZWF0b3JJRCI6InVwcmlkZSIsInVzZVZPRE9URkUiOmZhbHNlLCJmcm9tTWV6emFuaW5lIjpmYWxzZSwic3RvcmFnZVByb3ZpZGVyIjoyLCJ0cmFjayI6IjI5ODBlNzIyOTQ5NjkzOGQ4OGQyODEwZmRjNWM5ZWQ5IiwicmVuZGl0aW9uIjoiNDQ5MTM0NDQ5IiwibXV4aW5nIjoiNDk5ODMzNjE4In0&amp;s=w6XCjsKzw7fCmsKKwo_DoAvDq0LCniYHa39Tw6JEw6BUwojDiENvDMO9wqw7ccOwwpM" media="../../eaef9dea5159cf968be84241b5cedfe7/video/240/seg_$Number$.mp4?p=eyJ0eXBlIjoic2VnbWVudCIsInZpZGVvSUQiOiJlYWVmOWRlYTUxNTljZjk2OGJlODQyNDFiNWNlZGZlNyIsIm93bmVySUQiOjY2MzQxNTgsImNyZWF0b3JJRCI6InVwcmlkZSIsInNlZ21lbnREdXJhdGlvblNlY3MiOjQsInVzZVZPRE9URkUiOmZhbHNlLCJmcm9tTWV6emFuaW5lIjpmYWxzZSwic3RvcmFnZVByb3ZpZGVyIjoyLCJ0cmFjayI6IjI5ODBlNzIyOTQ5NjkzOGQ4OGQyODEwZmRjNWM5ZWQ5IiwicmVuZGl0aW9uIjoiNDQ5MTM0NDQ5IiwibXV4aW5nIjoiNDk5ODMzNjE4In0&amp;s=bcOaIsOHPV_Cu8KiwqppCzkvw70Uwp8XHTbDqztcY8KVfkg6wqIHS8Ktw54" startNumber="1" timescale="30000"></SegmentTemplate>
            </Representation>
            <Representation id="449134457" bandwidth="680674" codecs="avc1.4d401e" frameRate="30/1" height="360" width="640">
                <SegmentTemplate duration="120000" initialization="../../eaef9dea5159cf968be84241b5cedfe7/video/360/init.mp4?p=eyJ0eXBlIjoiaW5pdCIsInZpZGVvSUQiOiJlYWVmOWRlYTUxNTljZjk2OGJlODQyNDFiNWNlZGZlNyIsIm93bmVySUQiOjY2MzQxNTgsImNyZWF0b3JJRCI6InVwcmlkZSIsInVzZVZPRE9URkUiOmZhbHNlLCJmcm9tTWV6emFuaW5lIjpmYWxzZSwic3RvcmFnZVByb3ZpZGVyIjoyLCJ0cmFjayI6IjI5ODBlNzIyOTQ5NjkzOGQ4OGQyODEwZmRjNWM5ZWQ5IiwicmVuZGl0aW9uIjoiNDQ5MTM0NDU3IiwibXV4aW5nIjoiNDk5ODMzNjI2In0&amp;s=w5_DpDHCncOVwpVtw4HDnMOOwqfDl8Oyw6PDrMO1w5Y5PmlSJnUxFyHCq8KHwpbDqMO0w4M" media="../../eaef9dea5159cf968be84241b5cedfe7/video/360/seg_$Number$.mp4?p=eyJ0eXBlIjoic2VnbWVudCIsInZpZGVvSUQiOiJlYWVmOWRlYTUxNTljZjk2OGJlODQyNDFiNWNlZGZlNyIsIm93bmVySUQiOjY2MzQxNTgsImNyZWF0b3JJRCI6InVwcmlkZSIsInNlZ21lbnREdXJhdGlvblNlY3MiOjQsInVzZVZPRE9URkUiOmZhbHNlLCJmcm9tTWV6emFuaW5lIjpmYWxzZSwic3RvcmFnZVByb3ZpZGVyIjoyLCJ0cmFjayI6IjI5ODBlNzIyOTQ5NjkzOGQ4OGQyODEwZmRjNWM5ZWQ5IiwicmVuZGl0aW9uIjoiNDQ5MTM0NDU3IiwibXV4aW5nIjoiNDk5ODMzNjI2In0&amp;s=wp07RcKIN8ORB8Oww5fCkB02AMO2dEEgwqpRAhVcZMOywqbCnxhuwqDCqHrDrg" startNumber="1" timescale="30000"></SegmentTemplate>
            </Representation>
            <Representation id="449134468" bandwidth="1113178" codecs="avc1.4d401f" frameRate="30/1" height="480" width="854">
                <SegmentTemplate duration="120000" initialization="../../eaef9dea5159cf968be84241b5cedfe7/video/480/init.mp4?p=eyJ0eXBlIjoiaW5pdCIsInZpZGVvSUQiOiJlYWVmOWRlYTUxNTljZjk2OGJlODQyNDFiNWNlZGZlNyIsIm93bmVySUQiOjY2MzQxNTgsImNyZWF0b3JJRCI6InVwcmlkZSIsInVzZVZPRE9URkUiOmZhbHNlLCJmcm9tTWV6emFuaW5lIjpmYWxzZSwic3RvcmFnZVByb3ZpZGVyIjoyLCJ0cmFjayI6IjI5ODBlNzIyOTQ5NjkzOGQ4OGQyODEwZmRjNWM5ZWQ5IiwicmVuZGl0aW9uIjoiNDQ5MTM0NDY4IiwibXV4aW5nIjoiNDk5ODMzNjM3In0&amp;s=w4LCtMKUw5InVcKkRcKJw6dNw55_USATD8KCw5zDi0rCpMOyWjXDqsKAwqDCnQzDrMKd" media="../../eaef9dea5159cf968be84241b5cedfe7/video/480/seg_$Number$.mp4?p=eyJ0eXBlIjoic2VnbWVudCIsInZpZGVvSUQiOiJlYWVmOWRlYTUxNTljZjk2OGJlODQyNDFiNWNlZGZlNyIsIm93bmVySUQiOjY2MzQxNTgsImNyZWF0b3JJRCI6InVwcmlkZSIsInNlZ21lbnREdXJhdGlvblNlY3MiOjQsInVzZVZPRE9URkUiOmZhbHNlLCJmcm9tTWV6emFuaW5lIjpmYWxzZSwic3RvcmFnZVByb3ZpZGVyIjoyLCJ0cmFjayI6IjI5ODBlNzIyOTQ5NjkzOGQ4OGQyODEwZmRjNWM5ZWQ5IiwicmVuZGl0aW9uIjoiNDQ5MTM0NDY4IiwibXV4aW5nIjoiNDk5ODMzNjM3In0&amp;s=SBvCiTjDkMKfw7YTcsOeJcOhw6ltbEtMVXTCv8KGBBTCkFwUKsOyVFl2Bg" startNumber="1" timescale="30000"></SegmentTemplate>
            </Representation>
            <Representation id="449134513" bandwidth="2380128" codecs="avc1.4d401f" frameRate="30/1" height="720" width="1280">
                <SegmentTemplate duration="120000" initialization="../../eaef9dea5159cf968be84241b5cedfe7/video/720/init.mp4?p=eyJ0eXBlIjoiaW5pdCIsInZpZGVvSUQiOiJlYWVmOWRlYTUxNTljZjk2OGJlODQyNDFiNWNlZGZlNyIsIm93bmVySUQiOjY2MzQxNTgsImNyZWF0b3JJRCI6InVwcmlkZSIsInVzZVZPRE9URkUiOmZhbHNlLCJmcm9tTWV6emFuaW5lIjpmYWxzZSwic3RvcmFnZVByb3ZpZGVyIjoyLCJ0cmFjayI6IjI5ODBlNzIyOTQ5NjkzOGQ4OGQyODEwZmRjNWM5ZWQ5IiwicmVuZGl0aW9uIjoiNDQ5MTM0NTEzIiwibXV4aW5nIjoiNDk5ODMzNjgyIn0&amp;s=XcOPGjHClhTDn8KcBMKXw5HCo8ONIcKuQMKqbSp8wpLCnMOZXMODw6zDqMKew4_CoEpu" media="../../eaef9dea5159cf968be84241b5cedfe7/video/720/seg_$Number$.mp4?p=eyJ0eXBlIjoic2VnbWVudCIsInZpZGVvSUQiOiJlYWVmOWRlYTUxNTljZjk2OGJlODQyNDFiNWNlZGZlNyIsIm93bmVySUQiOjY2MzQxNTgsImNyZWF0b3JJRCI6InVwcmlkZSIsInNlZ21lbnREdXJhdGlvblNlY3MiOjQsInVzZVZPRE9URkUiOmZhbHNlLCJmcm9tTWV6emFuaW5lIjpmYWxzZSwic3RvcmFnZVByb3ZpZGVyIjoyLCJ0cmFjayI6IjI5ODBlNzIyOTQ5NjkzOGQ4OGQyODEwZmRjNWM5ZWQ5IiwicmVuZGl0aW9uIjoiNDQ5MTM0NTEzIiwibXV4aW5nIjoiNDk5ODMzNjgyIn0&amp;s=SMOOEMOcw5HDq8KTw67CqUMhYzVUGWobQXVvK8OYLMKdw4TDk8KMw5_DnV5ow48" startNumber="1" timescale="30000"></SegmentTemplate>
            </Representation>
        </AdaptationSet>
    </Period>
</MPD>```

**Comment by hugovdev** [comment]
Pull request to fix this issue #11478
